cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -4153,12 +4189,13 @@ search = | tstats `cs_summariesonly_authentication` count, min(_time) as firstTi
     [| stats sum(count) as count by index, app, src, user \
     | eval new_user = user." (".count.")" \
     | fields - count] \
-| stats sum(count) as count, min(firstTime) as firstTime, max(lastTime) as lastTime, values(new_*) as * by index, app, src \
+| stats sum(count) as count, min(firstTime) as firstTime, max(lastTime) as lastTime, values(is_privileged_user) as is_privileged_user, values(new_*) as * by index, app, src \
 | where count > `cs_authentication_excessive_vpn_login_failure_limit` \
 | `cs_human_readable_time_format(firstTime)` \
 | `cs_human_readable_time_format(lastTime)` \
 | eval reasons=mvjoin(signature, ", ") | fields - signature | eval users=mvjoin(user, ", ") | fields - user \
-| eval cyences_severity = case(count>1000, "critical", count>500, "high", count>100, "medium", true(), "low") \
+| eval cyences_severity = if(is_privileged_user=="Yes", case(count>500, "critical", count>100, "high", true(), "medium"), case(count>1000, "critical", count>500, "high", count>100, "medium", true(), "low")) \
```

Comment:
This is considered even if only one of the user is previledged right?

Reply:
Yes, correct. here, is_privileged_user field will be multivalued but search if one if the user is privileged from that.